route53_crossfeed_app.tf

```
@@ -226,13 +226,23 @@ resource "aws_route53_record" "crossfeed_staging_AAAA" {
 resource "aws_route53_record" "crossfeed_staging_CNAME" {
```

For consistency, this should become `crossfeed_staging_CNAME1` now that we are adding `crossfeed_staging_CNAME2`:
```suggestion
resource "aws_route53_record" "crossfeed_staging_CNAME1" {
```